app/models/folder.rb

```diff
@@ -10,26 +10,29 @@
 #  created_at  :datetime         not null
 #  updated_at  :datetime         not null
 #  team_id     :integer          default(0), not null
+#  personal_inbox :boolean       default(false)
 #
 
-#  Copyright (c) 2008-2017, Puzzle ITC GmbH. This file is part of
-#  Cryptopus and licensed under the Affero General Public License version 3 or later.
-#  See the COPYING file at the top-level directory or at
-#  https://github.com/puzzle/cryptopus.
-
 class Folder < ApplicationRecord
   belongs_to :team
-  has_many :encryptables, -> { order :name }, dependent: :destroy
+  has_many :encryptables, dependent: :destroy
```

Comment:
can we just remove default order here? this impacts all other teams too ... order by name seams pretty handy to me? you might add another order condition for the inbox folder to prioritize it?

` -> { order [:personal_inbox, :name] }`

Reply:
Lias told me it's unnecessary at this point since it gets overwritten anyway in Folder Serializer